### DISCUSSION TITLE
How to import anything?

### DISCUSSION THREAD

**Discussion by MeetMartin** [panva/jose#329] [Q&A] [answered]
Hi,

according to the repo https://github.com/panva/jose, I can import everything just by
```
import * as jose from 'jose'
```
But that leads to Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined

I found that I have to use imports like import { SignJWT } from 'jose/jwt/sign'; from Google. But I cannot find how these imports are structured from docs on GitHub. When I open https://github.com/panva/jose/blob/HEAD/docs/classes/jwt_sign.SignJWT.md#readme it does not tell me that I should import from 'jose/jwt/sign'

Is there some other documentation resource that I should be using?

Thank you for help.

**Comment by panva** [comment, marked as answer]
Is it possible you installed version 3 but are looking at docs from version 4?

install version 4, use supported node versions, follow docs in the readme.

**Reply by MeetMartin** [reply]
yes that was the issue, thank you for the help
